python/mlc_llm/serve/config.py

```diff
@@ -93,6 +93,11 @@ class GenerationConfig:  # pylint: disable=too-many-instance-attributes
 
     response_format : ResponseFormat
         The response format of the generation output.
+
+    pinned : bool
```

Comment:
add a subdataclass debug_config here. to include

- pin_system_prompt
- ignore_eos